docker/README.md

```diff
@@ -0,0 +1,38 @@
+# Pokémon Showdown Docker
+
+This Dockerfile will build an image that can be used to run a docker container with. This application needs runs on port 8000, so you need to bind that port to your host. If you want to use a custom config you will also need to supply your own `config.js`, an example of this can be found at `config/config-example.js` of this repository.
```

Comment:
```suggestion
This Dockerfile will build an image that can be used to run a docker container with. This application needs to run on port 8000, so you need to bind that port on your host. If you want to use a custom config, you will also need to supply your own `config.js`; an example of this can be found at `config/config-example.js` in this repository.
```